README.md

```diff
@@ -158,6 +159,19 @@ You can also join the mailing lists for the [UXL Foundation](https://lists.uxlfo
 
 - oneAPI, oneCCL and OFI: Path to Heterogeneous Architecure Programming with Scalable Collective Communications: [recording](https://www.youtube.com/watch?v=ksiZ90EtP98&feature=youtu.be) and [slides](https://www.openfabrics.org/wp-content/uploads/2020-workshop-presentations/502.-OFA-Virtual-Workshop-2020-oneCCL-v5.pdf)
 
+## Notice of Deprecation 
+
+### Notice of Deprecation Of Existing C++ API
```

Comment:
simplifying the title to "Deprecation of C++ API" or "Upcoming Deprecation of C++ API" ?

Reply:
Agreed.  Done.